app/Libraries/LocaleMeta.php

```diff
@@ -99,8 +99,8 @@ class LocaleMeta
             'name' => 'Tagalog',
             'flag' => 'PH',
         ],
-        'zh' => [
-            'name' => '中文',
+        'zh-CN' => [
```

Comment:
That's because jQuery-timeago use zh-CN so I use zh-CN(if not,timeago will use English)
Actually,"简体中文" is "zh-CN","繁體中文" is "zh-TW"
they are similar but some not,I'll ask them :sweat_smile: